pkgs/applications/window-managers/weston/default.nix

```diff
@@ -27,10 +27,10 @@ stdenv.mkDerivation rec {
   };
 
   postPatch = ''
-    # raise neatvnc version bound to 0.8.0
+    # raise neatvnc version bound to < 0.9.0
     # https://gitlab.freedesktop.org/wayland/weston/-/issues/890
     substituteInPlace libweston/backend-vnc/meson.build \
-      --replace-fail "'neatvnc', version: ['>= 0.7.0', '< 0.8.0']" "'neatvnc', version: ['>= 0.7.0', '<= 0.8.0']"
+      --replace-fail "'neatvnc', version: ['>= 0.7.0', '< 0.8.0']" "'neatvnc', version: ['>= 0.7.0', '< 0.9.0']"
```

Comment:
If it works this should probably be upstreamed (at least in an issue) again?  Since the linked issue is now closed because 0.8.0 support has been added.

Reply:
Not yet in a release version.

Reply:
Oh, I was misreading and thought we were upgrading to 0.9.0.  Never mind. :)